File: exercicios/java/Temperatura.java
/*
====================================
EXERCÍCIO 3 - CONVERSÃO DE TEMPERATURA
====================================

Peça uma temperatura em Celsius e
converta para Fahrenheit.
*/

import java.util.Scanner;

public class Temperatura {
    public static void main(String[] args) {

        Scanner entrada = new Scanner(System.in);

        System.out.print("Digite a temperatura em Celsius: ");

        // double -> temperatura em Celsius
        double celsius = entrada.nextDouble();

        // double -> temperatura convertida
        double fahrenheit = (celsius * 9 / 5) + 32;

        System.out.println("Temperatura em Fahrenheit: " + fahrenheit);

        entrada.close();
    }
}

File: exercicios/javascript/Temperatura.js
/*
====================================
EXERCÍCIO 3 - CONVERSÃO DE TEMPERATURA
====================================

Peça uma temperatura em Celsius e
converta para Fahrenheit.
*/

const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question("Digite a temperatura em Celsius: ", (resposta) => {
    // double -> temperatura em Celsius (convertida para número decimal)
    let celsius = parseFloat(resposta);

    // double -> temperatura convertida
    let fahrenheit = (celsius * 9 / 5) + 32;

    console.log("Temperatura em Fahrenheit: " + fahrenheit);

    readline.close();
});